android/src/main/java/com/reactlibrary/RNAppAuthModule.java

```diff
@@ -96,24 +109,53 @@ private WritableMap tokenResponseToMap(TokenResponse response) {
         return additionalParametersHash;
     }
 
+    private AppAuthConfiguration createAppAuthConfiguration(ConnectionBuilder connectionBuilder) {
+        return new AppAuthConfiguration
+                .Builder()
+                .setConnectionBuilder(connectionBuilder)
+                .build();
+    }
+
+    private ConnectionBuilder createConnectionBuilder(Boolean allowInsecureConnections) {
+        if (allowInsecureConnections.equals(true)) {
+            return UnsafeConnectionBuilder.INSTANCE;
+        }
+
+        return DefaultConnectionBuilder.INSTANCE;
+    }
+
+    private Uri buildConfigurationUriFromIssuer(Uri openIdConnectIssuerUri) {
```

Comment:
Since we are now using `fetchFromUrl` instead of `fetchFromIssuer`, we need to construct the issuer WKID endpoint ourselves. Annoyingly the method on `AuthorizationServiceConfiguration` was private, so I replicated it here. This should be a very stable method, since the URL format is driven by spec and not likely to change.